### ISSUE TITLE
Eigen Decompsition

### ISSUE THREAD

**Issue by Sagnik-Epigen** [open2c/inspectro#1]
I was trying the pipeline with HT1080 HI-C data (.mcool generated from distiller.nf). Unfortunately, eigdecomp rule never worked. I think it is the problem with partition. I am sending you my SRA code (SRR7586416, SRR7586418). Please tell me what to do.

**Comment by nvictus** [comment]
Hello. Can you post the traceback of the error you see?

**Comment by Sen0108** [comment]
Yeah sure... first one when I have to tried to run it in vscode individually each rule. Second one, is within the conda environment.

<img width="914" alt="Screen Shot 2022-07-12 at 11 03 13 AM" src="https://user-images.githubusercontent.com/97987859/178522172-3e781aa9-3a91-4c59-9abf-4021ef8ce76c.png">
<img width="517" alt="Screen Shot 2022-07-12 at 11 03 49 AM" src="https://user-images.githubusercontent.com/97987859/178522334-1256ebaa-8c71-4dd9-a4b6-c41dfb3abf53.png">
.

**Comment by Sen0108** [comment]
the assembly was hg38

**Comment by nvictus** [comment]
The chromosome order in your coolers might be different than what is expected.

What is the output of `cooler dump -t chroms {your_data}.mcool::resolutions/50000` ?

**Comment by Sen0108** [comment]
Ok. What is the solution? I am sending you the cooler dump output
[HT1080.txt](https://github.com/open2c/inspectro/files/9094481/HT1080.txt)

**Comment by Sen0108** [comment]
Need to go for sorting?

**Comment by nvictus** [comment]
Easiest thing is to rebuild your coolers from pairs using a properly sorted chromsizes file.

**Comment by Sen0108** [comment]
Ok. let me try that. Thanks.

**Comment by Sen0108** [comment]
I got a new error

<img width="1419" alt="Screen Shot 2022-07-12 at 1 12 07 PM" src="https://user-images.githubusercontent.com/97987859/178552409-a28a9786-fd6d-441a-a072-77f7e26e2ae3.png">

**Comment by nvictus** [comment]
Did you "look above for the error message" as it says?

**Comment by Sen0108** [comment]
Yeah but it's just the log like the previous image

**Comment by nvictus** [comment]
The log should contain somewhere the actual error that caused the workflow to terminate. You can browse the complete log file printed on the last line of what you posted. 

There's no way I can solve your issue without that information.

**Comment by Sen0108** [comment]
This is the complete log
<img width="1416" alt="Screen Shot 2022-07-12 at 2 16 46 PM" src="https://user-images.githubusercontent.com/97987859/178565642-23a60768-3629-47d4-ae4d-937e2d6a2786.png">

**Comment by nvictus** [comment]
Hmm. That's strange. Looks like it might be a resource problem. What are the specs on your machine, cores & memory?

**Comment by Sen0108** [comment]
I am working in cluster under conda environment. We can go as much as 40 vCPUs core and maximum RAM = 372 GB

**Comment by Sen0108** [comment]
I tried with another machine having specification 7 (11th Gen), GPU (RPX3060), RAM =16 GB. But the same problem occurred
